Timer: documentation about maxUnits prop (#746)
aragon/aragon-ui: src/components/Timer/README.md

Format output in years 'y', months 'M', days 'd', hours 'h', minutes 'm', seconds 's'

### `maxUnits`

| Type         | Default value |
| ------------ | ------------- |

> | `Number`    |       -1       |

Reviewer: ```suggestion
| `Number`     |       `-1`       |
```

> | `Number`     |       `-1`       |

Reviewer 2: Small nitpick: it looks like we could align this table

> Is the number of units to be displayed regardless of the `format`.
> By default is set to -1 which will dispay all the available untis, but if you set `format={'yMdhms'}` and `maxUnits={1}` it will ony display the number of years.

Reviewer 3: ```suggestion
Is the number of units to be displayed regardless of the `format`.
By default, it's set to -1 which will display all the available units. If it's set to a number n, which can be from 1 to all the units used in the `format` prop, it will display only the first n units, left to right.
```

> By default, it's set to -1 which will display all the available units. If it's set to a number n, which can be from 1 to all the units used in the `format` prop, it will display only the first n units, left to right.

Reviewer: ```suggestion
By default, it’s set to -1 which will display all the available units. If it’s set to a number `n`, which can be from `1` to all the units used in the `format` prop, it will display only the first `n` units, left to right.
```
